src/app/screens/sign/sign.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="container-register">
+  <div class="container-wolox">
```

Comment:
why do you use nested div? it's necessary?

Reply:
Because the first div is the view container and the second one is the form container, I did this more to organize the elements in the html.

Reply:
Aah okay. You are thinking ahead, right?